### DISCUSSION TITLE
Just follwing the "experience" route...

### DISCUSSION THREAD

**Discussion by nk8020** [valinet/ExplorerPatcher#911] [Ideas]
Latest reflections of Microsoft to its "ms experience" program members might seem familiar: they strongly asked for return of the ability to group start menu icons by dragging them on-top another.

Seems that after solving the taskbar location issue, that is the natural next step..

-Anybody feel the same way?

**Comment by valinet** [comment]
Huh?

**Comment by nk8020** [comment]
Sorry if I was not clear.
Maybe this will clarify my intent:
One of the useful features in the Win 10 start menu was the ability to group "pinned to start" icons, into groups (and even name the groups).
Probing to see what MS was doing in its "latest & greatest" win11 edition, I found that they actually restored this property to the start menu, but it will not appear in the win11 22000 version, at least until they officially decide to release an updated version.

I was wondering if this was possible to do with the ExplorerPatcher, and not wait for MS.
